apps/artist/routes.coffee

```diff
@@ -20,13 +20,16 @@ query = """
 module.exports.index = (req, res, next) ->
   artist = new Artist id: req.params.id
   artist.fetch
+    error: res.backboneError
     cache: true
     success: ->
       artist.maybeFetchAndSetFeaturedBio =>
-        res.locals.sd.ARTIST = artist.toJSON()
-        showAuctionLink = artist.get('display_auction_link')
-        res.render 'page', artist: artist, sort: req.query?.sort, showAuctionLink: showAuctionLink
-    error: res.backboneError
+        artist.fetchArtworks
```

Comment:
Yes that would be cool! ... I've heard this be called callback hell?  Sounds like something we should try to avoid